app/battle/battle_starter.py

```diff
@@ -0,0 +1,30 @@
+from app.knights.knight_creation import Knight
+
+
+class Battle:
+    def __init__(self, knights_config: dict) -> None:
+        self.knights = []
+        for name in knights_config.keys():
+            self.knights.append(Knight(knights_config[name]))
```

Comment:
You can use list comprehension here. If you think it looks messy, you can create another method for that